docs/cluster-configuration.md

```diff
@@ -159,6 +159,15 @@ the [dataImportCronTemplates field](#configure-custom-golden-images), even if th
 
 **Default**: `true`
 
+### deployTektonTaskResources Feature Gate
+Set the `deployTektonTaskResources` feature gate to true to allow Tekton Tasks operator to deploy its resources. TTO will 
```

Comment:
Suggestion: maybe we need to introduce the new TTO  acronym before start using it?

```suggestion
Set the `deployTektonTaskResources` feature gate to true to allow Tekton Tasks operator (TTO) to deploy its resources. TTO will 
```